docs/api/cards.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Cards"
+summary: |
+  Bulk endpoints for Cards.
+---
+
+# Cards
+
+> You can view live OpenAPI docs in your own running Metabase at `/api/docs`.
+   So if your Metabase is at https://www.your-metabase.com you could view
+   the API docs at https://www.your-metabase.com/api/docs.
+
+Bulk endpoints for Cards.
+
+## `POST /api/cards/dashboards`
+
+Get the dashboards that multiple cards appear in. The response is a sequence of maps, each of which has a `card_id`
+  and `dashboards`. `dashboard` may include an `:error` key, either `:unreadable-dashboard` or
+  `:unwritable-dashboard`. In the case of an `unreadable-dashboard` the dashboard details (name, ID) will NOT be
+  present.
+
+### PARAMS:
+
+-  **`card_ids`** sequence of value must be an integer greater than zero.
+
+## `POST /api/cards/move`
+
+Moves a number of Cards to a single collection or dashboard.
+
+  For now, just either succeed or fail as a batch - we can think more about error handling later down the road.
```

Comment:
This seems like an internal note that should be a code comment rather than a docstring